src/ancillary/multiprocess.ts

```diff
@@ -64,7 +64,10 @@ export class Multiprocess extends EventEmitter {
       this.work()
       return
     }
-    let processes = options.workers || cpus().length // TODO workers = -1 or undef means no workers
```

Comment:
Would be good to understand why `cpus()` is returning incorrect results in the first place.

Reply:
Otherwise this means having to constantly keep the env var updated when updating CAS resources in k8s.

Reply:
https://github.com/nodejs/node/issues/28762

Reply:
And changed the code to use settings from what's inside the Docker.

Reply:
Nice!! Thanks 😊